src/components/CoursePage/WhatYouWillLearn.js

```diff
@@ -16,30 +18,35 @@ const WhatYouWillLearn = ({ whatYouWillLearn, enrollBtnURL }) => {
 			<div className="text-white max-w-2xl mx-auto">
 				{whatYouWillLearn.map((info, index) => (
 					<div key={index}>
-						<Accordion>
+						<Accordion className="my-2">
```

Comment:
why is this called my-2? make it a more descriptive name, please.

Reply:
:neckbeard:

Reply:
hello